packages/@uppy/core/src/loggers.js

```diff
@@ -0,0 +1,21 @@
+const getTimeStamp = require('@uppy/utils/lib/getTimeStamp')
+
+// Swallow logs, default if logger is not set or debug: false
+const nullLogger = {
+  debug: (...args) => {},
+  warn: (...args) => {},
+  error: (...args) => {}
+}
+
+// Print logs to console with namespace + timestamp,
+// default if logger: 'debug' or debug: true
+const debugLogger = {
+  debug: (...args) => console.debug(`[Uppy] [${getTimeStamp()}]`, ...args),
```

Comment:
```suggestion
  debug: (...args) => {
    const debug = console.debug || console.log
    debug(`[Uppy] [${getTimeStamp()}]`, ...args)
  },
```
this should work hopefully

Reply:
Hmm, thanks! And then a note about this in the docs? 🤔

Reply:
If we're mentioning `console.debug` specifically somewhere, I guess :thinking: but otherwise I don't think that's really necessary

Reply:
Well, I thought we’d use our `debugLogger` as an example on the docs page, describing that you need to provide 3 methods in your custom logger. And if someone proxies those to `console` methods and stumbles into the same issue with IE, I think a note wouldn’t hurt :)

Reply:
:+1: